### PR TITLE
Jv 119 credits button placement

### LINKED ISSUE #119: Credits button misplacement
## Expected Behavior
Credits button should be realigning with  smaller screen sizes.

## Actual/Current Behavior
The button is getting pushed to the right when viewing on smaller screens.

### PR DESCRIPTION
**Pull request by Wolftownn**
## Changes
1. Realigns the credits button to its correct placement in mobile size screens.

## Purpose
Allows the user to access the credits button no matter what size screen they are using.

## Approach
Added a "fixed" CSS styling to the button for mobile screens.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #119

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Nice and easy fix. Good job!

File: pokemon-app/src/styles/CreditScreen.css [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
     border-radius: 50%;
-    top: 27.75%;
-    left: 48.75%;
+    top: 27.5%;
+    left: 48.5%;
     z-index: 10;
-    height: 1.5rem;
-    width: 1.5rem;
+    height: 1.75rem;
+    width: 1.75rem;
```
Comment (krferrell) [inline pokemon-app/src/styles/CreditScreen.css:8]: Small changes can make a big difference!

**Review by krferrell — APPROVED** [review]
Can't wait to see this credits page.

**Review by BlackCubes — APPROVED** [review]
Simple fix. Good job!

**Pull request merged**